### PR TITLE
Updated failing openssl.patch for Ruby 2.2.4 and 2.2.5

### LINKED ISSUE #3988: Failing openssl.patch for Ruby 2.2.4 and 2.2.5
Installation of ruby 2.2.4 and 2.2.5 fail due broken `openssl.patch`. Check conversation in https://github.com/rvm/rvm/issues/3831#issuecomment-288833298

### PR DESCRIPTION
**Pull request by pkuczynski**
Fixes #3988

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
 *	Architecture detection using uname instead of dpkg [\#3948](https://github.com/rvm/rvm/issues/3948)
 * Help section of the rvm.io page points to irc.freenode.net instead of github [\#3939](https://github.com/rvm/rvm/issues/3939)
 * Make sure stderr output is printed on console and not captured into log files (No ticket)
+* Failing openssl.patch for Ruby 2.2.4 and 2.2.5 [\#3988](https://github.com/rvm/rvm/issues/3988)
```
Comment (mpapis) [inline CHANGELOG.md]: `Remove`, `Fix` - what was the action of this commit ;)

Comment (pkuczynski) [inline CHANGELOG.md]: Both :) Newer version of ruby had already tests patched. Dont ask me why :)

Comment (mpapis) [inline CHANGELOG.md]: I mean - we should use language that describes the action, right now the message does not say anything about the action, someone can assume we are adding failing `openssl.patch` - and wonder `why someone would add a failing patch?` :)

Comment (pkuczynski) [inline CHANGELOG.md]: I already fixed the language :)

Comment (mpapis) [inline CHANGELOG.md]: my bad :D

**Pull request merged**